exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -248,6 +248,9 @@ drill.exec: {
     spooling: {
       delete: true,
       size: 100000000
+    },
+    unlimited: {
+      size: 500000000
```

Comment:
The name seems confusing. "Unlmited" means (except in American cell phone plans) that there is no limit. Yet, we see a limit with the `size`. Is this the `receive_buffer_size`?

Reply:
make sense